### DISCUSSION TITLE
Abnormal behavior with sublime 4

### DISCUSSION THREAD

**Discussion by git-amw** [Jatana/FastOlympicCoding#50] [General]
I'm experiencing  abnormal behavior since I've upgrade sublime to version 4. Whenever I try custom input, Test runner does not respond after hitting enter, it just goes to newline and end up in a infinite loop.

![subl](https://user-images.githubusercontent.com/74599802/121121591-7d8ec180-c83d-11eb-92a6-0e77826c9216.JPG)

**Comment by pwn2ooown** [comment]
Hmm, I have similar problem too. But for my computer, this bug appears while I use the Enter button beside the number section of the keyboard. When I use the "main" Enter of the keyboard, it becomes fine.
